tests/test_kfp_output.py

```diff
@@ -13,17 +14,20 @@ def test_kfp_writer_save_and_overwrite_metric():
 
 
 def test_serialize_and_write_metrics():
-    file_name = "metrics.json"
-    file_path = os.path.dirname(os.getcwd())
-    expected_data = {
-        "metrics": [
-            {"name": "mAR", "numberValue": 0.787, "format": "RAW"},
-            {"name": "mAP", "numberValue": 0.667, "format": "RAW"},
-        ]
-    }
-    writer_obj = KubeflowPipelineWriter(filename=file_name, filepath=file_path)
-    writer_obj.add_metric(name="mAR", val=0.787)
-    writer_obj.add_metric(name="mAP", val=0.667)
-    writer_obj.write_metric()
-    assert writer_obj.data == expected_data
-    assert os.path.exists(os.path.join(file_name, file_path))
+    with tempfile.TemporaryDirectory() as tmp:
+        file_name = "metrics.json"
+        file_path = tmp
+        expected_data = {
+            "metrics": [
+                {"name": "mAR", "numberValue": 0.787, "format": "RAW"},
+                {"name": "mAP", "numberValue": 0.667, "format": "RAW"},
+            ]
+        }
+        writer_obj = KubeflowPipelineWriter(
+            filename=file_name, filepath=file_path
+        )
+        writer_obj.add_metric(name="mAR", val=0.787)
+        writer_obj.add_metric(name="mAP", val=0.667)
```

Comment:
You can add two variables, `mar_value = 0.787` and `map_value = 0.667` and don't need to hard coded twice. Like `writer_obj.add_metric(name="mAR", val=mar_value) ...`